src/joysticks/Controller.java

```diff
@@ -0,0 +1,336 @@
+package joysticks;
+
+import edu.wpi.first.wpilibj.GenericHID;
+
+public class Controller extends GenericHID {
+
+    private final int BUTTON_DOWN;
+    private final int BUTTON_UP;
+    private final int BUTTON_LEFT;
+    private final int BUTTON_RIGHT;
+    private final int CENTER_RIGHT;
+    private final int CENTER_LEFT;
+    private final int STICK_LEFT;
+    private final int STICK_Right;
```

Comment:
Why not capital letters if all of the rest are capital?
Oh, you thought you got rid of my reviews? never :)😈